slackapi/slackbot-bolt.mjs

```diff
@@ -0,0 +1,312 @@
+import { BooksStore as books, UserStore as users } from '../models/books-store.mjs';
+import dotenv from 'dotenv';
+dotenv.config();
+import { default as Bolt } from '@slack/bolt';
+import { WebClient } from '@slack/web-api';
+
+const slackSigninSecret = process.env.SLACK_SIGIN_SECRET;
+const slackToken = process.env.SLACK_OAUTH_TOKEN;
+
+// Create a Bolt Receiver
+const receiver = new Bolt.ExpressReceiver({ signingSecret: slackSigninSecret });
+// Web Client
+const slackClient = new WebClient(slackToken);
+
+// Bolt App Server
+const slackApp = new Bolt.App({
+  token: slackToken,
+  receiver
+});
+
+// Slack event interactions
+slackApp.event('app_mention', async ({event, client}) => {
+  await client.chat.postMessage({ channel: event.channel, text: `Hello <@${event.user}>! `});
+});
+
+slackApp.command('/addbook', async ({ ack, body, client }) => {
+  await ack();
+
+  try {
+    const result = await client.views.open({
+      trigger_id: body.trigger_id,
```

Comment:
You could have the modal json in other file to make it thinner

Reply:
Sure, thanks for the comment Alejandro :D